### PR TITLE
3: Update intro

### LINKED ISSUE #3: Update intro slides
- [x] Links
- [ ] People
	- [x] Daniel
	- [x] Liming - @clarkliming please add to the branch
	- [ ] Joe - @shajoezhu please add to the branch
	- [x] Shuang @shuangli22 please add to the branch
- [x] chat channel - waiting for @shajoezhu information
- [x] Slido - does not work in China, at least in the office I cannot even create an account on the website. What would be an alternative @shajoezhu @clarkliming @stevemun ?
	- [x] would https://www.mentimeter.com/ work?

### PR DESCRIPTION
**Pull request by danielinteractive**
closes #3

### CODE REVIEW

File: slides/01_intro.qmd [hunk 1]
```diff
@@ -26,36 +26,46 @@ image: thumbnails/hello.jpg
...
+-   Feel free to connect at [`r fontawesome::fa("github")` Github](https://github.com/clarkliming "Liming GitHub")
+:::
+
+::: {.column width="30%"}
+![](resources/liming.jpg)
+:::
+:::
+
+## Joe
+
```
Comment (danielinteractive) [inline slides/01_intro.qmd:45]: @shajoezhu can you please push your bullets and photo onto this branch? thanks!

File: slides/01_intro.qmd [hunk 2]
```diff
@@ -26,36 +26,46 @@ image: thumbnails/hello.jpg
...
-    -   optimal clinical trial design
-    -   statistical genetics & high dimensional data
--   Methodology statistician with BI since '21
--   R-tist, extensive experience with R packages, version control, and high-performance computing
--   feel free to connect at `r fontawesome::fa("linkedin")` [LinkedIn](https://www.linkedin.com/in/kevin-kunzmann-6486a11bb/) or `r fontawesome::fa("github")` [Github](https://github.com/kkmann)
+-   Master in Pharmacology from Fudan University
+-   5-year experience in clinical data management, curation and related R tools development
+-   Technical Lead of several data FARIification R packages, co-auther of the Restrict Data Check Shiny App and the Raw Data Compare R Package
+-   Feel free to connect at [`r fontawesome::fa("github")` Github](https://github.com/shuangli22 "Shuang GitHub")
 :::
```
Comment (danielinteractive) [inline slides/01_intro.qmd:71]: @Stevemun if you like to have a slide please add one - if you prefer staying in background also ok of course, up to you

File: slides/01_intro.qmd [hunk 3]
```diff
@@ -26,36 +26,48 @@ image: thumbnails/hello.jpg
...
 
-## Kevin
+## Joe
 
 ::: columns
+::: {.column width="70%"}
+-   Ph.D. in Statistics
+-   Postdoc at the University of Oxford for 6 years, Data Scientist at Roche for the last 3 years, technical engineering lead for the NEST SME team.
+-   Multiple open-source packages on Github and CRAN, see [this page](https://shajoezhu.github.io/software/) for details.
+-   Feel free to connect at [`r fontawesome::fa("linkedin")` LinkedIn](http://www.linkedin.com/in/joe-zhu-464b5818 "Joe Zhu at LinkedIn") or [`r fontawesome::fa("github")` Github](https://github.com/shajoezhu "Joe Zhu at GitHub")
```
Comment (clarkliming) [inline slides/01_intro.qmd:51]: this link does not work

Comment (shajoezhu) [reply slides/01_intro.qmd:51]: the linkedin in or github?

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @danielinteractive

**Pull request merged**